group.yml

```
@@ -1,5 +1,5 @@
-name: oadp-1.5
-csv_namespace: oadp
+name: mta-8.0
+csv_namespace: mta
```

```suggestion
csv_namespace: mta
product: mta
```
ref: https://github.com/openshift-eng/ocp-build-data/blob/oadp-1.5/group.yml#L3